db/oracle_schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20181018082620) do
+ActiveRecord::Schema.define(version: 20181105220514) do
```

Comment:
I did this one manually because it was giving me this error otherwise:
```
ActiveRecord::StatementInvalid: OCIError: ORA-02429: cannot drop index used for enforcement of unique/primary key
```

Reply:
Help from Hery: [drop the constraint then drop the index](http://www.dba-oracle.com/t_ora_02429_cannot_drop_index.htm
)